### ISSUE TITLE
"Skip to content" link does not work on Fabric documentation page

### ISSUE THREAD

**Issue by citelao** [OfficeDev/office-ui-fabric-react#9563]
## Reproduce

1. Open [a page of Fabric component documentation](https://developer.microsoft.com/en-us/fabric#/controls/web/focuszone)
1. On page load, tab once until you see the "skip to main content" section appear.
1. Press enter.

## Actual behavior:

1. Navigates to a 404 page.

## Expected behavior:

1. Page scrolls to (& focus starts at) content.

**Comment by citelao** [comment]
I could not find the repo for the documentation webpage :).

**Comment by KevinTCoughlin** [comment]
> I could not find the repo for the documentation webpage :).

Thanks for the report @citelao. We'll take a look and provide a fix!

Can repro via https://developer.microsoft.com/en-us/fabric#mainContent

**Comment by ecraig12345** [comment]
@Jahnp @jordandrako @kenotron FYI (add this to the list of reasons for moving to our own site...)

**Comment by micahgodbolt** [comment]
we're blocked on this issue because the main content link is part of the mwf shell. I reached out to the team to see what we can do to resolve.

**Comment by ecraig12345** [comment]
I actually checked in a workaround for this awhile back (but didn't know about this issue)--just adding CSS to hide that element. While it would be nice to fix the root cause, at least the broken experience is gone, so I think it's okay to close this now.

**Comment by ecraig12345** [comment]
Oops, misunderstanding. We actually hid a different element.

**Comment by ecraig12345** [comment]
A couple possible solutions here:

1. Hide this element with CSS too
2. Remove the element's href and add an onClick which jumps to the right place

Either of those is kind of hacky, but I think option 2 is better because it preserves the original intended accessibility benefits of that element. The tricky part is figuring out exactly when/where to run that code.

**Comment by citelao** [comment]
Thanks!
